src/sdk/lsf-sdk.js

```diff
@@ -335,8 +335,14 @@ export class LSFWrapper {
       this.lsf.resetAnnotationStore();
     }
 
+    // Initial idea to show counter for Manual assignment only
+    // But currently we decided to hide it for any stream
+    // const distribution = this.project.assignment_settings.label_stream_task_distribution;
+    // const isManuallyAssigned = distribution === "assigned_only";
+
     // undefined or true for backward compatibility
     this.lsf.toggleInterface("postpone", this.task.allow_postpone !== false);
+    this.lsf.toggleInterface("topbar:task-counter", !isFF(FF_DEV_3734));
```

Comment:
This reads a bit confusing, we disabling this for only when the FF is off? So when this FF does go on later, does that mean it should work for all streams? I don't see the correlation with this FF, for clarity it would probably be better to just set to false explicitly for now since it is misleading if those assumptions are true.

Reply:
fixed ff name to avoid confusion